tests/src/JIT/HardwareIntrinsics/X86/dir.props

```diff
@@ -1,3 +1,7 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props"/>
+
+  <ItemGroup>
+    <ProjectReference Include="../../../../Common/CoreCLRTestLibrary/CoreCLRTestLibrary.csproj" />
+  </ItemGroup>
```

Comment:
Since you are adding the project reference globally for all HardwareIntrinsics tests can you please remove the same lines from all the individual project files (e.g. from Avx_r.csproj Avx_ro.csproj etc)

Reply:
@echesakov, do you have any issue with me doing that in the follow up PR?

I am fixing all templated tests to use `Generator.Get*` right now and can include it as part of that

Reply:
@tannergooding No issues, that would be fine